### ISSUE TITLE
`docusaurus-plugin-typedoc` causes fatal error on Windows

### ISSUE THREAD

**Issue by mysticatea** [tgreyuk/typedoc-plugin-markdown#245]
Hello.

First, thank you for the useful plugins.

After I upgraded `docusaurus-plugin-typedoc`, it started to fail on Windows.

```
Error while parsing Markdown frontmatter.
This can happen if you use special characters in frontmatter values (try using double quotes around that value).
Can't process doc metadatas for doc at path "C:\Users\***\index.md" in version "current"
Loading of version failed for version "current"
YAMLException: unknown escape sequence at line 4, column 14:
    slug: "/util\foo"
                 ^
```

It looks that `docusaurus-plugin-typedoc` uses Windows paths as-is for the slug.

https://github.com/tgreyuk/typedoc-plugin-markdown/blob/0047faf1d3e2ec91752cef0603f2838ce1e70be9/packages/docusaurus-plugin-typedoc/src/front-matter.ts#L56

**Comment by tgreyuk** [comment]
thanks - fix in docusaurus-plugin-typedoc@0.15.3

**Comment by mysticatea** [comment]
I have confirmed that's fixed. Thank you!
